setup.py

```diff
@@ -21,7 +21,10 @@
     url='https://github.com/nuodb/nuodb-python',
     license='BSD License',
     long_description=open(readme).read(),
-    install_requires=['pytz>=2015.4'],
+    install_requires=[
+        'pytz>=2015.4',
+        'cryptography>=2.6.1'
+    ],
```

Comment:
Do you need to add those packages to the NuoDB installation?

Reply:
cryptography is already packed with NuoDB. Not sure about pytz, but it is not newly added (just looks like that because of the change in formatting).

Reply:
```
$ nuopython -m pip freeze | grep pytz
pytz==2018.5
```

Reply:
If we allow for a fallback do we still use install_requires?  Is there something like "install_recommends" or something?

Maybe not worthwhile, I don't know.